crypto/src/lib.rs

```diff
@@ -39,14 +39,18 @@ use ursa::{
     },
 };
 
+// Hiding constants is a bad idea. For one, you're forcing the user to
+// create temporaries, but also you're not actually hiding any
+// information that can be used in malicious ways. If you want to hide
+// these, I'd prefer inlining them instead.
```

Comment:
ok, user was thought to be encouraged to access them through `Algorithm::to_string` just to reduce the API surface are. `ToString` does crate a temporary string, that's true

Reply:
I was reckless, now I see, the strings are not the same

Reply:
> I was reckless, now I see the strings are not the same

Correction, it's all good, the strings are matching. No harm in making them public, though

Reply:
The string representations aren't guaranteed to be the same. 

Additionally, the whole point of having these as global constants is that every place where they have to appear in is automatically updated once the variables themselves are changed. Making them private, particularly considering that they don't store any privileged information is counter-productive.